### PR TITLE
fix lint issues

### PR DESCRIPTION
**Pull request by pavelpikta**
* fix lint issues
* add aws direct tests
* update role requirements

### CODE REVIEW

File: tasks/create_domain.yml [hunk 1]
```diff
@@ -87,7 +87,7 @@
     mode: "0555"
   become: True
   when: |
-    node_manager.install == True and
+    node_manager.install | bool and
```
Comment (tgadiev) [inline tasks/create_domain.yml]: Change conditional to short version with version filter

File: tasks/install-wls.yml [hunk 2]
```diff
@@ -13,6 +13,10 @@
       package:
         name: "{{ package }}"
         state: "present"
+      register: status
+      retries: 10
+      delay: 2
+      until: status is success
       loop: "{{ wls_requirements }}"
```
Comment (tgadiev) [inline tasks/install-wls.yml]: Why loop?

File: tasks/install-wls.yml [hunk 3]
```diff
@@ -72,8 +76,8 @@
       command: "{{ download_path }}/{{ wls_installer.files[0] }} \
       -mode=silent -silent_xml={{ wls_silent_install.dest }}"
       become_user: "{{ wls_user }}"
+      become: True
 
-  become: True
   when: |
     wls_installer.files[0].split(".")[-1] == "bin" and
     wls_version.split('.')[0] | int <= 10
```
Comment (tgadiev) [inline tasks/install-wls.yml]: Make conditional more readable
```yaml
when:
  - wls_installer.files[0].split(".")[-1] == "bin"
  - wls_version is version('10.0.0', '<=')
```

**Review by tgadiev — CHANGES_REQUESTED** [review]

**Review by tgadiev — APPROVED** [review]

File: tasks/install-wls.yml (hunk 3 above)
Comment (pavelpikta) [reply tasks/install-wls.yml]: Resolve in dd3eaa8

File: tasks/install-wls.yml (hunk 2 above)
Comment (pavelpikta) [reply tasks/install-wls.yml]: Resolve in d1a05d4

File: requirements.yml [hunk 4]
```diff
@@ -1,3 +1,2 @@
 ---
 - src: "lean_delivery.java"
-  version: 1.5
```
Comment (AlekseiSidorenko) [inline requirements.yml:3]: Is it OK, that we are downloading the latest java in this case?

Comment (AlekseiSidorenko) [reply requirements.yml:3]: please check the following compatibility matrix
https://docs.oracle.com/middleware/12212/wls/INTRO/compatibility.htm#INTRO112

Comment (AlekseiSidorenko) [reply requirements.yml:3]: As far as I know, WLS 12.x.x is able to work with 1.8 java, but WLS 10.3.6 can't, only with 1.7

Comment (AlekseiSidorenko) [reply requirements.yml:3]: We checked with Pavel that all is fine

**Review by AlekseiSidorenko — APPROVED** [review]

File: requirements.yml (hunk 4 above)
Comment (pavelpikta) [reply requirements.yml:3]: In molecule tests for WLS 10.3.6
```yaml
- role: lean_delivery.java
  java_major_version: 6
  java_minor_version: 45
```

**Pull request merged**